pkg/apis/core/validation/validation.go

```diff
@@ -412,14 +412,8 @@ func validateVolumeSource(source *core.VolumeSource, fldPath *field.Path, volNam
 	allErrs := field.ErrorList{}
 	if source.EmptyDir != nil {
 		numVolumes++
-		if !utilfeature.DefaultFeatureGate.Enabled(features.LocalStorageCapacityIsolation) {
```

Comment:
I just noticed there are four more places `utilfeature.DefaultFeatureGate.Enabled(features.LocalStorageCapacityIsolation)` is checked in validation and "disabled by feature-gate" errors returned. Those should be dropped.

Reply:
Done